tests/vmcheck/test-layering-relabel.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+#
+# Copyright (C) 2017 Red Hat Inc.
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place - Suite 330,
+# Boston, MA 02111-1307, USA.
+
+set -euo pipefail
+
+. ${commondir}/libtest.sh
+. ${commondir}/libvm.sh
+
+set -x
+
+# make sure there's nothing yet
+vm_assert_status_jq \
+  '.deployments[0]["base-checksum"]|not'
+
+# check that pkgs that install sepolicies have their changes take effect
+vm_build_selinux_rpm foobar-selinux /usr/bin/foobar install_exec_t
+vm_build_rpm foobar requires foobar-selinux
```

Comment:
There ends up being a global namespace for these RPMs right?  I'd vote we name them after the tests like `test-relabel` and `test-relabel-selinux` or something.

Reply:
No, RPMs built in a test are visible during that test only since they're part of the tmpdir we set up in `libtest.sh`. On the VM side, we also have a crude hack in `libvm.sh` that basically `rm -rf /tmp/vmcheck` to force a resync from scratch each time we build and RPM.